### PR TITLE
Fixed Iris default build/Updated Dev Guide

### PR DESCRIPTION
**Pull request by kurtis-lew**

### CODE REVIEW

File: app/boards/shields/iris/iris.keymap [hunk 1]
```diff
@@ -1,6 +1,3 @@
-# Copyright (c) 2020 Pete Johanson, Kurtis Lew
```
Comment (petejohanson) [inline app/boards/shields/iris/iris.keymap:1]: We should keep the copyright header, just use valid comments for DT. I believe this needs `/*  */` style comments.

File: docs/docs/dev-guide-new-shield.md [hunk 2]
```diff
@@ -18,6 +18,9 @@ The high level steps are:
 - Add a default keymap, which users can override in their own configs as needed.
 - Add support for features such as encoders, OLED displays, or RGB underglow.
 
+
```
Comment (petejohanson) [inline docs/docs/dev-guide-new-shield.md:21]: There shouldn't be any extra blank lines between list items

**Review by petejohanson — CHANGES_REQUESTED** [review]
A few minor items. Thanks!

**Comment by kurtis-lew** [comment]
@petejohanson addressed the things you wanted me to. Thanks!

File: docs/docs/dev-guide-new-shield.md [hunk 3]
```diff
@@ -302,3 +303,36 @@ and then flash with:
...
+          - corne_right
+          - kyria_left
+          - kyria_right
+          - lily58_left
+          - lily58_right
+          - iris_left
+          - iris_right
+          - romac
+		  - <MY_BOARD>
+		  - <MY_SPLIT_BOARD_left>
```
Comment (petejohanson) [inline docs/docs/dev-guide-new-shield.md:330]: Missed this the first go round, looks like some indent/whitespace inconsistency here.

**Review by petejohanson — COMMENTED** [review]
One minor item left.

File: docs/docs/dev-guide-new-shield.md (hunk 3 above)
Comment (kurtis-lew) [reply docs/docs/dev-guide-new-shield.md]: Just fixed the indents. Hopefully, that should be it.

**Review by petejohanson — APPROVED** [review]
LGTM! Thanks.

**Pull request merged**